src/memory_tracker.h

```diff
@@ -199,6 +199,10 @@ class MemoryTracker {
   inline void TrackField(const char* edge_name,
                          const std::basic_string<T>& value,
                          const char* node_name = nullptr);
+  template <typename T>
+  inline void TrackField(const char* edge_name,
+                         const std::basic_string_view<T>& value,
```

Comment:
A string view is not a memory retainer. I think this should not be added in MemoryTracker.

Reply:
No but the underlying thing it is a view of, which in this case we have no other way to access, *IS* a memory retainer and the view is what is giving us a view of that. I see no reason at all to block this PR on this point.

Reply:
Removed the changes to memory tracking entirely.

Reply:
I think a better way to add this info could be https://github.com/nodejs/node/pull/56881, wdyt?